pangeo_forge_recipes/config.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+
+from pangeo_forge_recipes.storage import CacheFSSpecTarget, FSSpecTarget
+from pangeo_forge_recipes.transforms import RequiredAtRuntimeDefault
+
+
+@dataclass
+class Config:
```

Comment:
Let's add this to the API Reference section of the docs.

And possibly link to it in either/or:

- The discussion of "Deploy-time configurable keyword arguments" under **Recipe composition > Transforms"
- a new Advanced Topics section, e.g. "Custom Transforms"

Reply:
thanks 💯 I'll get around to this after hearing back from Yuvi and other folks